lib/src/types/entity.dart

```diff
@@ -699,12 +699,20 @@ class AssetEntity {
 
   /// The mime type of the asset.
   ///  * Android: `MediaStore.MediaColumns.MIME_TYPE`.
-  ///  * iOS/macOS: Always null.
+  ///  * iOS/macOS: Always null. Use the async getter [mimeTypeAsync] instead.
   ///
   /// See also:
   ///  * https://developer.android.com/reference/android/provider/MediaStore.MediaColumns#MIME_TYPE
```

Comment:
Add doc references for both methods.

```suggestion
  ///  * [mimeTypeAsync] which is the asynchronized getter of the MIME type.
  ///  * https://developer.android.com/reference/android/provider/MediaStore.MediaColumns#MIME_TYPE
```